### PR TITLE
fix(editor): Fix design system form element sizing

### PR DESCRIPTION
**Pull request by cstuncsik**
## Summary

Tag size is not correctly passed down through form input components.

image

## Related Linear tickets, Github issues, and Community forum posts

[PAY-2034](https://linear.app/n8n/issue/PAY-2034/fix-design-system-form-components)

## Review / Merge checklist

- [ ] PR title and summary are descriptive.
- [ ] Tests included.
- [ ] PR Labeled with `release/backport`

### CODE REVIEW

File: packages/design-system/src/__tests__/render.ts
Comment (r00gm) [inline packages/design-system/src/__tests__/render.ts:1]: i think this file and `packages/design-system/src/__tests__/setup.ts` should be in a separate PR to don't mix them with the actual fix, wdyt?

File: packages/design-system/src/components/N8nFormInputs/FormInputs.vue [hunk 1]
```diff
@@ -13,7 +13,6 @@ export type FormInputsProps = {
 	columnView?: boolean;
 	verticalSpacing?: '' | 'xs' | 's' | 'm' | 'l' | 'xl';
 	teleported?: boolean;
-	tagSize?: 'small' | 'medium';
```
Comment (r00gm) [inline packages/design-system/src/components/N8nFormInputs/FormInputs.vue:16]: is this intentiional?

Comment (cstuncsik) [reply packages/design-system/src/components/N8nFormInputs/FormInputs.vue:16]: Yes, it's just a wrapper component and actually that caused sizing not working properly.
Sizes should be defined in the inputs array on which this component iterates and renders `FormInput.vue` components

File: packages/design-system/src/__tests__/render.ts
Comment (cstuncsik) [reply packages/design-system/src/__tests__/render.ts:1]: Sure, here you are
https://github.com/n8n-io/n8n/pull/11064

**Review by r00gm — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.63.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.63.0)

**Comment by janober** [comment]
Got released with [`n8n@1.63.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.63.0)
